minitests/graph_folding/Makefile

```diff
@@ -14,7 +14,6 @@ ${DATABASE}: build_node_lookup.py node_lookup.py | build_${PART}
 		--part ${PART} \
 		${DATABASE}
 
-<<<<<<< HEAD
```

Comment:
This should be squashed into the commit that introduced it, leaving broken commits like this causes bisecting to be even more of a pain than it already is.

Reply:
Thanks, for pointing that out. It should be fixed now.

Reply:
Maybe you forgot to push? The broken commits are still there.